pyca/pyca.cc

```diff
@@ -731,6 +731,9 @@ extern "C" {
     // Initialize python module
     DECLARE_INIT(pyca)
     {
+        // libca creates non-python threads so we must ensure that python threading is initialized
+        PyEval_InitThreads();
```

Comment:
This is already called below, before the first epics-related call. Is there a special need to call it earlier?
https://github.com/slaclab/pyca/blob/5e18525fcd2b39c68c5d8dd4b543d4d8e12dd5f2/pyca/pyca.cc#L791-L793